client-js/app/scripts/app.ts

```diff
@@ -187,7 +183,13 @@ window.addEventListener('WebComponentsReady', () => {
       });
     }
 
+    onNewItem(x) {
```

Comment:
since the parameter is not used, I'd suggest removing it. I used the name `x` in the callbacks elsewhere only because the variable was being discarded (`observable.add(x => doSomethingWithoutX())`). A better name would have been something like "ignored", but in any case I'd suggest just omitting it. If you do want to pass it through, however, then maybe choose a more meaningful name and include the type here.